doc/source/locale/zh_CN/LC_MESSAGES/development/xinference_internals.po

```diff
@@ -0,0 +1,554 @@
+msgstr ""
+"**定义 Actor 的行为**：每个 actor 都需要定义某些动作或行为来完成特定任务。"
+"例如，模型推理 ``WorkerActor`` 需要启动模型（``launch_model``）、"
+"列出该 actor 中的模型（``list_models``）、终止模型（``termininate_model``）。"
+"有两个特殊方法值得注意。``__post_create__`` 在创建 actor 之前调用，以便进行必要的初始化。"
+"而 ``__pre_destroy__`` 会在 actor 被销毁后调用，以便执行清理或最终任务。"
+
+#: ../../source/development/xinference_internals.rst:136
+msgid ""
+"**Reference Actor and Invoke Methods**: When an actor is created, it "
+"yields a reference variable so that other actors can reference it. The "
+"actor reference can also be referenced with the address. Suppose the "
+"``WorkerActor`` is created and the reference variable is ``worker_ref``,"
+"  the ``launch_model`` method of this actor class can be invoked by "
+"calling ``worker_ref.launch_model()``."
+msgstr ""
+"**引用actor和调用方法**：当创建一个 actor 时，它会产生一个引用变量，"
+"以便其他角色可以引用它。actor 引用也可以用地址来引用。假设创建了 ``WorkerActor``，"
+"且引用变量为 ``worker_ref``，那么就可以通过调用 ``worker_ref.launch_model()`` 来调用"
+"该 actor 类的 ``launch_model``。"
+
+#: ../../source/development/xinference_internals.rst:141
+msgid ""
+"**Inference Engine**: The actor can manage the process, and the inference"
+" engine is also a process. In the launch model part of the "
+"``WorkerActor``, we can initialize different inference engines according "
+"to the user's need. Therefore, Xinference can support multiple inference "
+"engines and can easily adapt to new inference engines in the future."
+msgstr ""
+"**推理引擎**：actor 可以管理进程，而推理引擎也是一进程。在 ``WorkerActor`` 的启动模型部分，"
+"我们可以根据用户的需要初始化不同的推理引擎。因此，Xinference 可以支持多种推理引擎，并能轻松适应未来的新推理引擎。"
+
+#: ../../source/development/xinference_internals.rst:146
+msgid ""
+"See `Xoscar document <https://xoscar.dev/en/latest/getting_started/llm-"
+"inference.html>`_ for more actor use cases."
+msgstr ""
+"请参阅 `Xoscar 文档 <https://xoscar.dev/en/latest/getting_started/llm-inference.html>`_ 了解更多 actor 用例。"
+
+#: ../../source/development/xinference_internals.rst:149
+msgid "Concurrency"
+msgstr "并发性"
+
+#: ../../source/development/xinference_internals.rst:150
+msgid ""
+"Both Xinference and Xoscar highly utilize coroutine programming of "
+"``asyncio``."
+msgstr ""
+"Xinference 和 Xoscar 都高度利用了 ``asyncio`` 进行协程编程"
+
+#: ../../source/development/xinference_internals.rst:152
+msgid ""
+"If you're not familiar with Pythons's ``asyncio``, you can see more "
+"tutorials for help:"
+msgstr ""
+"如果您不熟悉 Python 的 ``asyncio``，可以查看更多教程以获得帮助："
+
+#: ../../source/development/xinference_internals.rst:154
+msgid ""
+"`Real Python's asyncio Tutorial <https://realpython.com/async-io-"
+"python/>`__"
+msgstr ""
+
+#: ../../source/development/xinference_internals.rst:156
+msgid ""
+"`Python Official Documentation "
+"<https://docs.python.org/3/library/asyncio.html>`__"
+msgstr ""
+
+#: ../../source/development/xinference_internals.rst:159
+msgid "Model"
+msgstr "模型"
+
+#: ../../source/development/xinference_internals.rst:160
+msgid ""
+"Xinference supports different types of models including large language "
```

Comment:
This translation did not take effect, see https://inference--1178.org.readthedocs.build/zh-cn/1178/development/xinference_internals.html

Reply:
@qinxuye fix this issue. 
Check if there is something else to add?